next.config.js

```diff
@@ -1,6 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
+  // basePath: "./src",
```

Comment:
- [ ] Don't commit commented code to master.

Reply:
changed.